libs/barista-components/chart/src/selection-area/selection-area.ts

```diff
@@ -189,18 +193,12 @@ export class DtChartSelectionArea implements AfterContentInit, OnDestroy {
     private _changeDetectorRef: ChangeDetectorRef,
     // tslint:disable-next-line: no-any
     @Inject(DOCUMENT) private _document: any,
-    @Optional() private _viewportResizer: DtViewportResizer,
-  ) {}
+    @Optional() public _viewportResizer: DtViewportResizer,
```

Comment:
Do we want this in the breaking-changes list, as we do in the filter-field range, and make it mandatory in v9.0.0?

Reply:
I've added the comment here as well, thx for noticing!